subjects/ai/nlp-scraper/audit/README.md

```diff
@@ -22,15 +22,15 @@ project
 
 ###### Does the structure of the project look like the above?
 
-###### Does the readme file give an introduction of the project, show the username, describe the feature engineering and show the best score on the leaderboard?
+###### Does the README file give an introduction of the project, show the username, describe the feature engineering and show the best score on the leaderboard?
 
 ###### Does the environment contain all libraries used and their versions that are necessary to run the code?
 
-##### Scrapper
+##### Scraper
 
 ##### There are at least 300 news articles stored in the file system or the database.
 
-###### Run the scrapper with `python scrapper_news.py` and fetch 3 documents. The scrapper is not expected to fetch 3 documents and stop by itself, you can stop it manually. does it run without any error and store the 3 files as expected?
+###### Run the scraper with `python scraper_news.py` and fetch 3 documents. The scraper is not expected to fetch 3 documents and stop by itself, you can stop it manually. Does it run without any error and store the 3 files as expected?
```

Comment:
this question should be like that 

`##### Run the scraper with `python scraper_news.py` and fetch 3 documents. The scraper is not expected to fetch 3 documents and stop by itself, you can stop it manually.`
`###### Does it run without any error and store the 3 files as expected?`